doc/md/fundamentals/2-types/3-functions.md

```diff
@@ -198,7 +208,7 @@ The `getBalance` function has function type `shared query () -> async Nat`.
 A good example of a composite query might be a bank that holds references to its individual accounts, implemented as separate actors, and provides a composite query that sums the deposits in all its accounts:  
 
 ```motoko no-repl
-actor Counter {
+persistent actor Counter {
```

Comment:
The indentation in this example is messed up (but I can't fix it with a suggestion)